doc/source/installation/apple_arm.rst

```diff
@@ -26,25 +26,7 @@ To install the dependencies (mpi, p4est, trilinos and METIS) all together using
 
-To ensure that the Lethe test suite works, deal.II must be configured with p4est version 2.3.6. In the subfolder ``deal.II-toolchain/packages/``, open the ``p4est.package`` file with a text editor and change the following lines:
-
-+--------+------------------------------------------------+-------------------------------------------------------------------------------+
-| line # | initial line                                   | changed line                                                                  |
-+========+================================================+===============================================================================+
-|     9  | ``VERSION=2.3.2``                              | ``#VERSION=2.3.2``                                                            |
-+--------+------------------------------------------------+-------------------------------------------------------------------------------+
-|     10 | ``CHECKSUM=076df9e...``                        | ``#CHECKSUM=076df9e...``                                                      |
-+--------+------------------------------------------------+-------------------------------------------------------------------------------+
-|     11 | ``CHECKSUM="${CHECKSUM} b41c8ef29ca...``       | ``#CHECKSUM="${CHECKSUM} b41c8ef29ca...``                                     |
-+--------+------------------------------------------------+-------------------------------------------------------------------------------+
-|     12 | ``CHECKSUM="${CHECKSUM} 0ea6e4806b6...``       | ``#CHECKSUM="${CHECKSUM} 0ea6e4806b6...``                                     |
-+--------+------------------------------------------------+-------------------------------------------------------------------------------+
-|     13 |                                                | .. code-block:: text                                                          |
-|        |                                                |   :class: copy-button                                                         |
-|        |                                                |                                                                               |
-|        |                                                |   VERSION=2.3.6                                                               |
-|        |                                                |   CHECKSUM=4b35d9cc374e3b05cd29c552070940124f04af8f8e5e01ff046e39833de5e153   |
-+--------+------------------------------------------------+-------------------------------------------------------------------------------+
+To ensure that the Lethe test suite works, deal.II must be configured with p4est version 2.3.6 which is the current default candi version of p4est. Otherwise, application tests which include restart files will fail.
```

Comment:
```suggestion
To ensure that the Lethe test suite works, deal.II must be configured with p4est version 2.3.6, the current default candi version of p4est. Otherwise, application tests that include restart files will fail.
```